### PR TITLE
[fix]fixed dangerous usage of namespace std

### PR DESCRIPTION
**Pull request by zclllyybb**
According to [namespace.std] and many practices(like [CppCoreGuidelines](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#sf7-dont-write-using-namespace-at-global-scope-in-a-header-file) and [Don’t reopen namespace std](https://quuxplusone.github.io/blog/2021/10/27/dont-reopen-namespace-std/)), reopening `namespace std` or `using namespace std` is UB or very dangerous. Fixed them.

# Proposed changes

Issue Number: close #xxx

## Problem Summary

Describe your changes.

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [x] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [x] No
    - [ ] No Need
3. Has the document been added or modified:
    - [ ] Yes
    - [x] No
    - [ ] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [x] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [x] No

## Further comments

Still some similar behavior stays in some code-generators.

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zclllyybb** [comment]
您好，您的邮件我已收到。
这是我的自动回复。
                                  ***@***.***

**Comment by HappenLee** [comment]
Hello，Thanks for your PR。 Please modify the PR titile for the check

**Review by Yukang-Lian — DISMISSED** [review]
Nice work! Thanks for your contribution. LGTM.

File: be/src/olap/aggregate_func.cpp [hunk 1]
```diff
@@ -17,11 +17,10 @@
 
 #include "olap/aggregate_func.h"
 
-namespace std {
-namespace {
-// algorithm from boost: http://www.boost.org/doc/libs/1_61_0/doc/html/hash/reference.html#boost.hash_combine
+/// algorithm from boost: http://www.boost.org/doc/libs/1_61_0/doc/html/hash/reference.html#boost.hash_combine
```
Comment (HappenLee) [inline be/src/olap/aggregate_func.cpp:20]: keep the origin comment “//” not "///"

File: be/src/util/cityhash102/city.cc [hunk 2]
```diff
@@ -33,8 +33,7 @@
 #include <algorithm>
 #include <string.h>  // for memcpy and memset
 
-using namespace std;
-
+//using namespace std;
```
Comment (HappenLee) [inline be/src/util/cityhash102/city.cc:36]: just del the useless code, do not comment

Comment (zclllyybb) [reply be/src/util/cityhash102/city.cc]: thx for the reminder. fixed them.

**Review by Yukang-Lian — APPROVED** [review]
LGTM

**Review by BiteTheDDDDt — APPROVED** [review]
LGTM

**Pull request merged**